django_cloud_deploy/cli/prompt.py

```diff
@@ -1075,6 +1075,31 @@ def _validate(self, s: str):
                               '"test@example.com"').format(s))
 
 
+class TakeSurveyPrompt(TemplatePrompt):
+    """Ask the user whether they would like to take a survey."""
+
+    SURVEY_LINK = 'https://google.qualtrics.com/jfe/form/SV_3wwUubKBJnC7Fxr'
+
+    def prompt(self, console: io.IO):
+        """Ask users to take a survey.
+
+        If the user would like to take the survey, then the method will open
+        their web browser and direct them to an existing Qualtics survery.
+
+        Args:
+            console: Object to use for user I/O.
+            step: Message to present to user regarding what step they are on.
```

Comment:
You don't have any of these...does this class actually follow the contract for being a prompt? It might make sense for this to not be a prompt?

Reply:
I made this class not inheriting TemplatePrompt